extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for Algebraic Expression Input
+ * interaction
+ */
+
+var objects = require(process.cwd() + '/extensions/objects/protractor.js');
+
+var customizeInteraction = function() {
+  // There are no customizations.
+};
+
+var expectInteractionDetailsToMatch = async function(elem) {
```

Comment:
What does this check do in other interactions? Is it to test the customization args for the interaction (which in your case, there is none of)? Since this test seems to check just the presence.

Reply:
As mentioned [here](https://github.com/oppia/oppia/wiki/Creating-Interactions), this function is supposed to verify that the interaction is displayed correctly. This is why I've checked if the interaction element is present or not. Also, this is how it's done for other interactions as well.

Reply:
Ah ok.